### PR TITLE
Community demo extended example

### PR DESCRIPTION
**Pull request by nebril**
This change is [Reviewable](https://reviewable.io/reviews/mirantis/k8s-appcontroller/57)

### CODE REVIEW

File: examples/extended/create.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ kubectl.sh create -f existing_job.yaml
 
 kubectl.sh create -f ../../manifests/appcontroller.yaml
 
-sleep 10
+sleep 17
```
Comment (gluke77) [inline examples/extended/create.sh]: "It's a kind of magic.. It's a knid of magic..."

Comment (gluke77) [inline examples/extended/create.sh]: Sleep Driven Development :)

Comment (pigmej) [inline examples/extended/create.sh]: Can't we pool k8s api instead?

Or at least, maybe could we add comment there?

**Comment by pigmej** [comment]
:lgtm:

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/mirantis/k8s-appcontroller/57#-:-KQ5g4XiN0D0TGN-ZgeF:bnfp4nl)*

File: examples/extended/create.sh [hunk 2]
```diff
@@ -5,7 +5,8 @@ kubectl.sh create -f existing_job.yaml
 
 kubectl.sh create -f ../../manifests/appcontroller.yaml
 
-sleep 10
+#wait for appcontroller pod creation
+sleep 20
```
Comment (gluke77) [inline examples/extended/create.sh]: For me on minikube with downloading fresh image from DockerHub this takes up to 2 minutes. Mb, we should be smarter or more flexible here. We could merge it now, but should think of it anyway.

Comment (pigmej) [inline examples/extended/create.sh]: Yeah, as I said in previous comment, we should probably query k8s api for required resources.

Comment (nebril) [inline examples/extended/create.sh]: OR we can put a docker pull here to pull the latest version of the image. I will do just that.

**Pull request merged**